chat-plugins/COMMANDS.md

```diff
@@ -0,0 +1,270 @@
+Command API
+===========
+
+Defining a Command
+------------------
+
+You can define the commands such as 'whois', then use it by typing /whois 
+into Pokemon Showdown.
+
+A command can be in the form:
+    ip: 'whois',
+This is called an alias: it makes it so /ip does the same thing as /whois.
+
+But to actually define a command, it's a function:
+
+    avatars: function (target, room, user) {
+        if (!this.canBroadcast()) return;
+        this.sendReplyBox('You can <button name="avatars">change ' +
+            'your avatar</button> by clicking on it in the <button ' +
+            'name="openOptions"><i class="icon-cog"></i> Options' +
+            '</button> menu in the upper right. Custom avatars are ' +
+            'only obtainable by staff.');
+    }
+
+	
+Parameters
+----------
+	
+Commands are actually passed five parameters:
+    function (target, room, user, connection, cmd, message)
+
+Most of the time, you only need the first three, though.
+
+    target = the part of the message after the command
+
+    room = the room object the message was sent to
+        The room name is room.id
+
+    user = the user object that sent the message
+        The user's name is user.name
+
+    connection = the connection that the message was sent from
+
+    cmd = the name of the command
+
+    message = the entire message sent by the user
```

Comment:
Everything I tried with this section looked gross. Any ideas on how to organize it better?